staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission/v1alpha1/register.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// GroupName is the group name use in this package
+const GroupName = "webhook.admission.apiserver.k8s.io"
```

Comment:
Against our group name inflation: `config.admission.apiserver.k8s.io`.

Reply:
> Against our group name inflation: config.admission.apiserver.k8s.io.

I'd be ok with that if there were a way to add to a scheme.  Then `admission.apiserver.k8s.io` with specific types like `WebhookConfiguration` makes sense.  I guess I'm building up my own scheme, so I could choose my favorite types and include those.

@liggitt preference?  The common case would still work (that's what I'm doing) and the advanced case could be made to work by a really clever person.

Reply:
There was a big discussion about this, I thought the judgement was xxx.config.k8s.io, under the assumption that you can look at ".config.k8s.io" and *know* that it's not a "real" api object served by the system.

Reply:
> There was a big discussion about this, I thought the judgement was xxx.config.k8s.io, under the assumption that you can look at ".config.k8s.io" and know that it's not a "real" api object served by the system.

Ok.  The existing apiserver config file group doesn't match (it predated the others), but I'm fine adjusting this here.